apps/web/src/app/(verify-email)/layout.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import Image from 'next/image';
+
+import backgroundPattern from '~/assets/background-pattern.png';
+
+type VerifyEmailLayoutProps = {
+  children: React.ReactNode;
+};
+
+export default function VerifyEmailLayout({ children }: VerifyEmailLayoutProps) {
+  return (
+    <main className="bg-sand-100 relative flex min-h-screen flex-col items-center justify-center overflow-hidden  px-4 py-12 md:p-12 lg:p-24">
```

Comment:
Are we still using `bg-sand`? I thought we were up to `bg-background`, I could be wrong though

Reply:
I took it from the `unauthenticated` layout. But will update it to use `bg-background`.

Reply:
That may mean it’s a me problem, feel free to leave it for now